docs/docs/examples/workflow/parallel_execution.ipynb

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
<p>If easy to add, I think might be beneficial to include a Workflow that doesn't set <code>num_workers</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>